crates/router/src/utils.rs

```diff
@@ -123,13 +123,34 @@ impl<E> ConnectorResponseExt
     }
 }
 
+pub fn is_zero_decimal_currency(currency: diesel_models::enums::Currency) -> bool {
```

Comment:
Same here, move inside Currency implementation in `hyperswitch_oss/crates/common_enums/src/enums.rs`

Reply:
Let's take it in separate pr